54.md

```diff
@@ -0,0 +1,60 @@
+NIP-54
+======
+
+Topical Notes
+-------------
+
+`draft` `optional`
+
+This NIP enables authors to post exclusively into topics, without polluting their main feed. Authors can post to locations, hashtags, labels, and urls in a way that only followers of those topics will receive their posts. 
+
+## Topical Note Kind
+
+Kind `21` defines a `kind:1`-equivalent event that MUST NOT be displayed unless the user follows one of the `g`, `t`, `r` or `l` tags. 
+
+```jsonc
+{
+  "kind": 21,
+  "tags": [
+    ["g", "<geohash>"],
+    ["t", "<hashtag>"],
+    ["r", "<url>"],
+    //.. 
+    ["L", "<label namespace>"],
+    ["l", "<label>", "<label namespace>"],
```

Comment:
Can we be a bit more explicit that only 1 of these tags is required?

Is it possible that one would ever use 2 or more?

Reply:
Yes, they can do one or more if you consider translations and spellings, like: 

```jsonc
{
  "kind": 21,
  "tags": [
    ["t", "Amethyst"],
    ["t", "amethyst"],
    ["t", "Ametista"],
    ["t", "ametista"],
  ],
  //...
}
```